base/challenge-skeleton/challenge/config/nsjail.cfg

```diff
@@ -38,6 +38,11 @@ mount: [
     src: "/config"
     dst: "/config"
     is_bind: true
+  },
+  {
+    src: "/pow-bypass"
+    dst: "/pow-bypass"
+    is_bind: true
   }
```

Comment:
this is ugly, since we're exposing infrastructure details to the challenge directory.
An alternative for a future change could be to use an init container to copy the pow-bypass directory over or to have k8s mount to /chroot/pow-byass directly